elections.md

```diff
@@ -1,55 +1,36 @@
-# Kubernetes Steering Committee Elections 
+# Kubernetes Steering Committee Elections
 
-This document outlines the process, for steering committee elections.  
+This document outlines the process, for steering committee elections.
 
-**NOTE:** This document is stale and requires reevaluation prior to next election cycle. 
 
-## Elections
-
-Special note: The bootstrap committee pledges to recuse itself from any direct
-election activities while they serve as continuity members. Members of the
-bootstrap committee will refrain from endorsing or otherwise advocating for any
-candidate (with the exception that the members of the bootstrap committee may
-vote in the elections, and may choose to run in the 2019 election).
-
-### Members of Standing
+### Eligibility for voting
 
-Initial standing in the Kubernetes community will be defined by the union of:
-* SIG leads
-* Approvers and reviewers in any Kubernetes owned repositories
-* Anyone with write access to a Kubernetes owned repository
+Eligibility to vote for steering committee members will be defined by:
 
-The bootstrap committee *explicitly* believes that this heuristic will be
-inaccurate and not represent the entire community. An exception form will be
-provided for people who have contributed to the project but may not meet these
-criteria. Exception applications will be approved by the bootstrap committee
-will vote on this eligibility, with simple majority determining eligibility.
-The exception process will be used as data points for refining the criteria for
-the future.
+* People who have at least 50 [contributions to the Kubernetes project over the past year as defined by devstats](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All) as of 2018-08-15
+* People who have submitted the [voting exception form] and are accepted by
```

Comment:
Last year, the bootstrap committee decided this.  This year, I'm suggesting this should be election officers' duty.  

Do others feel like steering committee should be deciding exceptions?

Reply:
SGTM.